### PR TITLE
Improve cloud environment setup commands

### PR DESCRIPTION
**Pull request by CsSatter**

### CODE REVIEW

File: import.sh [hunk 1]
```diff
@@ -33,12 +33,12 @@ mysql -u$MYSQL_USER -p$MYSQL_PASSWORD $MYSQL_DATABASE_NAME < $DIRECTORY/structur
...
-        VERSION="m2"
-        MAGERUN_COMMAND=$MAGERUN2_COMMAND
+if [ -f "$DIRECTORY/app/etc/local.xml" ]; then
+        VERSION="m1"
+        MAGERUN_COMMAND=$MAGERUN1_COMMAND
 else
-		VERSION="m1"
-		MAGERUN_COMMAND=$MAGERUN1_COMMAND
+		VERSION="m2"
+		MAGERUN_COMMAND=$MAGERUN2_COMMAND
```
Comment (lewisvoncken) [inline import.sh:41]: @CsSatter why this switch?

File: init_cloud_env.sh [hunk 2]
```diff
@@ -56,42 +52,31 @@ if [ "$VERSION" ]; then
...
-. $SCRIPTPATH/src/update_settings.sh
-
-. $SCRIPTPATH/src/secure_domain.sh
+## Move data.sql & structure.sql to $DIRECTORY
+cp $DOMAINS_PATH/structure.sql $DIRECTORY/structure.sql
+cp $DOMAINS_PATH/data.sql $DIRECTORY/data.sql
 
-. $SCRIPTPATH/src/nfs.sh
\ No newline at end of file
+. $SCRIPTPATH/update.sh "$NAME"
```
Comment (lewisvoncken) [inline init_cloud_env.sh:82]: @CsSatter why not nfs?

File: update.sh [hunk 3]
```diff
@@ -36,12 +36,12 @@ mysql -u$MYSQL_USER -p$MYSQL_PASSWORD $MYSQL_DATABASE_NAME < $DIRECTORY/structur
...
-        VERSION="m2"
-        MAGERUN_COMMAND=$MAGERUN2_COMMAND
+if [ -f "$DIRECTORY/app/etc/local.xml" ]; then
+        VERSION="m1"
+        MAGERUN_COMMAND=$MAGERUN1_COMMAND
 else
-	VERSION="m1"
-	MAGERUN_COMMAND=$MAGERUN1_COMMAND
+	VERSION="m2"
+	MAGERUN_COMMAND=$MAGERUN2_COMMAND
```
Comment (lewisvoncken) [inline update.sh:44]: @CsSatter why this change?

File: init_cloud_env.sh (hunk 2 above)
Comment (CsSatter) [reply init_cloud_env.sh:82]: update.sh runs nfs.sh :)

File: import.sh (hunk 1 above)
Comment (CsSatter) [reply import.sh:41]: This is because when you build a Magento Cloud environment locally it'll not provide a env.php.

Because we mostly work with Magento 2 projects I felt like it was warranted to make Magento 2 the default and check if app/etc/local.xml exists to change the project to Magento 1.

This way our tools will create a workable app/etc/env.php as well.

File: update.sh (hunk 3 above)
Comment (CsSatter) [reply update.sh:44]: See https://github.com/experius/Magento-2-Bash-Localhost-Installation-Script/pull/36#discussion_r435737657
